Samples/EmbeddedAuthWithSDKs/EmbeddedAuth/Signin/View Model/FormRows.swift

```diff
@@ -199,20 +199,24 @@ extension IDXClient.Remediation {
         case .challengePoll:
             return "Verify"
             
+        case .unlockAccount: fallthrough
```

Comment:
What about `case .unlockAccount, .selectAuthenticatorUnlockAccount`?

I see the major pros in this - you shouldn't care about previous line so if you re-arrange cases you don't miss it by chance.